ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js

```diff
@@ -163,28 +137,28 @@ export default class DepositEtherModal extends Component {
             })}
             {this.renderRow({
               logo: (
-                <div
+                <img
+                  alt=""
                   className="deposit-ether-modal__logo"
+                  src="./images/deposit-eth.svg"
                   style={{
-                    backgroundImage: "url('./images/coinswitch_logo.png')",
```

Comment:
Could you remove this logo as well? This was the only place it was used

Reply:
@Gudahtt Good catch. Just added a new commit to delete the logo as well.